packages/manager/modules/pci/src/projects/project/storages/databases/database/users/show-secret/show-secret.routing.js

```diff
@@ -15,7 +15,8 @@ export default /* @ngInject */ ($stateProvider) => {
       layout: 'modal',
       resolve: {
         user: /* @ngInject */ ($transition$) => $transition$.params().user,
-        type: /* @ngInject */ ($transition$) => $transition$.params().type,
+        type: /* @ngInject */ ($transition$) =>
+          $transition$.params().secretType,
```

Comment:
**nitpick:** Use the same variable to avoid any confusion
```suggestion
        secretType: /* @ngInject */ ($transition$) =>
          $transition$.params().secretType,
```

Reply:
Done here: https://github.com/ovh/manager/pull/10254/commits/996978e00ec605341c2b3b168439eb7d1c477ee3